Packages/manifest.json

```diff
@@ -1,13 +1,18 @@
 {
   "dependencies": {
-    "com.playeveryware.eos": "file:com.playeveryware.eos-4.1.0.tgz",
+    "com.playeveryware.eos": "https://github.com/EOS-Contrib/eos_plugin_for_unity_upm.git",
```

Comment:
This is the important change actually. Everything else we should skip (changes in metas, proj update). We will update projects to 2022 sometime in the future